samples/drivers/i2c/rtio_loopback/README.rst

```diff
@@ -0,0 +1,50 @@
+.. zephyr:code-sample:: i2c-rtio-loopback
+   :name: I2C RTIO loopback
+   :relevant-api: rtio
+
+   Demo I2C transfers between I2C controller and custom I2C target using RTIO.
```

Comment:
apologies if it comes across as a nit but, for consistency with other samples:

```suggestion
   Perform I2C transfers between I2C controller and custom I2C target using RTIO.
```

See https://github.com/zephyrproject-rtos/zephyr/blob/a2ac676a106a757fe4cb2664bab35ea985da95f9/doc/templates/sample.tmpl#L5-L6

Reply:
applied